README.md

```diff
@@ -3235,7 +3234,7 @@ CollectionView, make Instagram Discover within minutes.
 * [Brian Advent youtube channel](https://www.youtube.com/channel/UCysEngjfeIYapEER9K8aikw/videos) - Swift tutorials Youtube Channel.
 * [raywenderlich.com](https://www.raywenderlich.com/ios) - Tutorials for developers and gamers
 * [Mike Ash](https://www.mikeash.com/pyblog/)
-* [Big Nerd Ranch](https://www.bignerdranch.com/blog/categories/ios/)
+* [Big Nerd Ranch](https://www.bignerdranch.com/wp-content/uploads/2019/05/ios.jpg)
```

Comment:
I think we should remove this link. Isn't a tutorial website anymore.

Reply:
Sorry, I found the correct link of the tutorials blog: https://www.bignerdranch.com/blog/category/ios/

Reply:
Yes, sure! I even don't know where I found that weird link... :-)